### DISCUSSION TITLE
DFU on Jumper T-Lite V2

### DISCUSSION THREAD

**Discussion by Shlipez** [iNavFlight/inav#10500] [Q&A]
Please help me, I don't understand how to turn on the DFU on Jumper T-Lite V2, it says on the Internet to turn off the controller, hold down some horizontal buttons, press the on button and turn off the controller, I tried to do this, nothing worked, although a sign appeared on the screen with the inscription: ,,Key stuck."

**Comment by mmosca** [comment]
I am sorry, but I don't see how this is related to INAV.

Have you tried contacting customer support for Jumper, or asking around in the edgetx discord?

**Reply by Shlipez** [reply]
Sorry, I was just looking for a fpv community where someone might know about my problem.
